### ISSUE TITLE
Change completion direction when top-down(Near cursor)

### ISSUE THREAD

**Issue by xzbdmw** [zed-industries/zed#11200]
### Check for existing issues

- [X] Completed

### Describe the feature

When completion menu opens above current line, show selection of completion menu near cursor.

### If applicable, add mockups / screenshots to help present your vision of the feature

https://github.com/hrsh7th/nvim-cmp/wiki/Menu-Appearance#custom-menu-direction

**Comment by JosephTLyons** [comment]
Hey @xzbdmw, can you show a screenshot of what you are running into?

**Comment by xzbdmw** [comment]
Yeah, I want the selected entry always near my cursor, I think it is nature to human eyes.
Currently zed always select the top entry no matter completion direction.

https://github.com/zed-industries/zed/assets/97848247/b128750e-cef7-4ac5-8139-009d02343eda

**Comment by notpeter** [comment]
So you invert the order of the completion list so the "best" / "first" selection appears at the bottom of the list closest to the cursor. I like it.
